src/packages/popup/types.ts

```
@@ -0,0 +1,24 @@
+  closeIconPosition: string
+  closeIcon: ReactNode
+  left?: ReactNode
+  title?: ReactNode
+  description?: ReactNode
+  destroyOnClose: boolean
+  portal: Teleport
+  overlay: boolean
+  round: boolean
+  onOpen: () => void
+  onClose: () => void
+  onOverlayClick: (e: TouchEvent) => boolean | void
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)